tests/test-iso-kargs.sh

```diff
@@ -25,6 +26,14 @@ fi
 # Test all the modification operations.
 coreos-installer iso kargs modify -a foobar=oldval -a dodo "${iso}"
 coreos-installer iso kargs show "${iso}" | grep -q 'foobar=oldval dodo'
+# Stream modification to stdout.
+coreos-installer iso kargs modify -a foobar=oldval -a dodo -o - "${iso}" > "${stdout_iso}"
```

Comment:
This is modifying the same ISO that was just modified, so the kargs are already present.

What you could do instead is hash the stdout version before the one that modifies in-place. E.g.:

```
# Test all the modification operations
stdout_hash=$(coreos-installer iso kargs modify -a foobar=oldval -a dodo "${iso}" -o - | sha256sum)
coreos-installer iso kargs modify -a foobar=oldval -a dodo "${iso}"
coreos-installer iso kargs show "${iso}" | grep -q 'foobar=oldval dodo'

hash=$(sha256sum < ${hash})
if [ "${stdout_hash}" != "${hash}" ]; then
    fatal "Streamed hash doesn't match modified hash: ${stdout_hash} vs ${hash}"
fi
```

Reply:
The current tests are reasonable as is, but note that @jlebon's code above verifies something that the tests currently don't: that `modify` in stdout mode makes the same changes that regular `modify` makes.  All we're confirming right now is that `modify` and `remove` are symmetric, and that `show` is happy with the result.